src/prettytable/prettytable.py

```diff
@@ -1431,7 +1432,7 @@ def add_rows(self, rows):
         for row in rows:
             self.add_row(row)
 
-    def add_row(self, row):
+    def add_row(self, row, divider=False):
```

Comment:
Because it can be a bit unclear what Boolean means in `add_row(row, True)` or `add_row(row, False)`, shall we use a keyword argument (aka named parameter) for this new one?

```suggestion
    def add_row(self, row, *, divider=False):
```

Then we need to explicitly do `add_row(row, divider=True)` or `add_row(row, divider=False)` instead, which is much clearer.